src/main/python/ayab/tests/test_communication.py

```diff
@@ -90,7 +90,7 @@ def test_cnf_line(self):
         lineData = [0x23, 0x24]
         flags = 0x12
         crc8 = 0x57
-        self.comm_dummy.cnf_line(lineNumber, lineData, flags, crc8)
+        self.comm_dummy.cnf_line(lineNumber, lineData, flags)
         byte_array = bytearray([0xc0, 0x42])
```

Comment:
I calculate the checksum as 0x88.

Reply:
We have some discrepancy here.
@t0mpr1c3  has used a vanilla crc8 implementation, whereas @Sturla22 has used the Maxim algorithm for ayab-firmware. These algorithms yield different results for the same input vector. You can compare the outputs at https://crccalc.com/

The vector used by Sturla in the ayab-firmware test (https://github.com/AllYarnsAreBeautiful/ayab-firmware/blob/v1.0-dev/test/test_serial_encoding.cpp#L51) yields 0xE9 with CRC8/maxim, and yields 0x3D with vanilla CRC8.
So we have to change one of the algorithms ;) Who wants to? :)

```
0x42 0x00 0x00 0xde 0xad 0xbe 0xef 0x00  0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00
0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00  0x00 0x00 0x00 0x01
```

Reply:
I should match Sturla's, I suppose! Apart from anything else, there are fewer constraints on the desktop software.